pkg/uhttp/transport.go

```diff
@@ -16,6 +17,12 @@ import (
 	"golang.org/x/oauth2"
 )
 
+var authorizationRegexFilter *regexp.Regexp
```

Comment:
`var authorizationRegexFilter = regexp.MustCompile()`

Reply:
Going from compile to must compile 👍🏼